src/show-comments/show-general-comments.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+// @jsx h
+
+/* eslint-disable operator-linebreak */
+
+import { h } from 'dom-chef'
+import SelectorObserver from 'selector-observer'
+
+import './show-comments.css'
+
+export let stateShowComments = true
+
+const onClick = e => {
+    stateShowComments = !stateShowComments
+    if (stateShowComments) {
+        e.currentTarget.classList.remove('__rbb_comments_hidden')
+    } else {
+        e.currentTarget.classList.add('__rbb_comments_hidden')
+    }
+
+    const container = e.currentTarget.closest('#general-comments')
+    ;[...container.querySelectorAll('#comments-list li.comment')].forEach(
+        comment => {
+            comment.style.display = stateShowComments ? '' : 'none'
+        }
+    )
+}
+
+export default function insertShowGeneralComments(section: HTMLElement) {
```

Comment:
The code in this file looks extremely similar to the other one. I wonder if it makes sense to extract out the differing parts, provide them through arguments and re-use the common ones so that we only have one file to maintain for both buttons. What do you think?